src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(a) {
+      this.result += a;
+
+      return this;
+    },
+    subtract(a) {
+      this.result -= a;
+
+      return this;
+    },
+    multiply(a) {
+      this.result *= a;
+
+      return this;
+    },
+    divide(a) {
+      this.result /= a;
+
+      return this;
+    },
```

Comment:
You need to add a check for 0, because by the rules of mathematics, you cannot divide by 0